api.bs

```diff
@@ -2016,6 +2026,220 @@ To <dfn noexport>parse a `Save-Impression` header</dfn> given a [=header value=]
+1.  If |dict|["<code>[=measure-conversion/impression-callers=]</code>"] [=map/exists=]:
+     1.  Let |impressionCallers| be its [=map/value=].
+     1.  If |impressionCallers| is not an [=structured header/inner list=], or if any of
+         |impressionCallers|' [=list/items=] is not a [=structured header/string=],
+         return an error.
+     1.  Set |opts|.{{AttributionConversionOptions/impressionCallers}} to |impressionCallers|.
+1.  If |dict|["<code>[=measure-conversion/credit=]</code>"] [=map/exists=]:
+     1.  Let |credit| be its [=map/value=].
+     1.  If |credit| is not an [=structured header/inner list=], or if any of
+         |credit|'s [=list/items=] is not a [=structured header/decimal=] or
+         [=structured header/integer=], return an error.
+     1.  Set |opts|.{{AttributionConversionOptions/credit}} to |credit|.
+1.  If |dict|["<code>[=measure-conversion/value=]</code>"] [=map/exists=]:
+     1.  Let |value| be its [=map/value=].
+     1.  If |value| is not a positive [=structured header/integer=] in the
+         [=32-bit unsigned integer=] range, return an error.
+     1.  Set |opts|.{{AttributionConversionOptions/value}} to |value|.
+1.  If |dict|["<code>[=measure-conversion/max-value=]</code>"] [=map/exists=]:
+     1.  Let |maxValue| be its [=map/value=].
+     1.  If |maxValue| is not a positive [=structured header/integer=] in the
+         [=32-bit unsigned integer=] range, return an error.
+     1.  Set |opts|.{{AttributionConversionOptions/maxValue}} to |maxValue|.
+1.  Return (|opts|, |reportUrl|).
+
+Issue: Should we allow `http` for |reportUrl|'s [=url/scheme=]?
+Related to https://github.com/w3c/attribution/issues/146.
+
+</div>
+
+<div algorithm>
+To <dfn noexport>send a report</dfn> given a [=byte sequence=] |report|,
+a [=URL=] |url|, and an [=environment settings object=] |settings|:
+
+1. [=Assert=]: |url| is a [=potentially trustworthy URL=].
+1. [=Assert=]: |url|'s [=url/scheme=] is "`https`".
+1. Let |headers| be a new [=header list=] containing a [=header=] named `"Content-Type"` whose value is `"application/octet-stream"`.
+1. Let |request| be a new [=request=] with the following properties:
+    :   [=request/method=]
+    ::  "`POST`"
+    :   [=request/URL=]
+    ::  |url|
+    :   [=request/header list=]
+    ::  |headers|
+    :   [=request/body=]
+    ::  |report|
+    :   [=request/client=]
+    ::  |settings|
+    :   [=request/mode=]
+    ::  "`cors`"
+    :   [=request/cache mode=]
+    ::  "`no-store`"
+    :   [=request/keepalive=]
+    ::  true
+    :   [=request/credentials mode=]
+    ::  "`omit`"
+1. [=Fetch=] |request|, optionally retrying in the event of an error.
+
+Issue: Should the request use a more specific content-type?
+
+Issue: Should the request's [=request/service-workers mode=] be "`none`" instead
+of the default of "`all`"?
+
+Issue: Should the request's [=request/unsafe-request flag=] be set?
```

Comment:
This is largely moot.  We're using POST with something other than application/x-www-form-urlencoded, so we'll be doing a preflight for any report.  We can't really avoid doing that, and nor should we try.  Enterprises might not like us pushing these reports at random servers, even though the opportunity for attack seems very much remote (no site can really control the content of reports).